mkdocs.yml

```diff
@@ -258,7 +258,8 @@ plugins:
               - "!^_"
               - "^__init__$"
               - "!^can_replace$"
-
+          import:
+            - https://docs.python.org/3/objects.inv
```

Comment:
This allows cross-linking to the standard library docs; used here https://github.com/Textualize/textual/pull/1541/files#diff-681b255c8a47359bc04668c27ab9770e5326a66af7931e4e267e8e205cb43aeeR148